app/views/templates/PaymentsHistoryChargeHelper.scala

```diff
@@ -41,7 +41,18 @@ object PaymentsHistoryChargeHelper {
     "paymentsHistory.officerAssessmentChargeTitle",
     "paymentsHistory.officerAssessmentDebitChargeDescription")
 
-  val values = Seq(VatReturnDebitCharge, VatReturnCreditCharge, VatOfficerAssessmentCreditCharge, VatOfficerAssessmentDebitCharge)
+  object VatCentralAssessment extends PaymentsHistoryChargeHelper(
+    FinancialTransactionsConstants.vatCentralAssessment,
+    "paymentsHistory.vatCentralAssessmentTitle",
+    "paymentsHistory.vatCentralAssessmentDescription", "")
+
+  object VatDefaultSurcharge extends PaymentsHistoryChargeHelper(
+    FinancialTransactionsConstants.vatDefaultSurcharge,
+    "paymentsHistory.vatDefaultSurchargeTitle",
+    "paymentsHistory.vatDefaultSurchargeDescription", "")
```

Comment:
No need to add the empty string at the end. It has been defaulted in the helper.